tutorials/export/exporting_pcks.rst

```diff
@@ -51,6 +60,23 @@ PCK files usually contain, but are not limited to:
 The PCK files can even be an entirely different Godot project, which the
 original game loads in at runtime.
 
+.. _doc_exporting_pcks_about_mods_using_pck_files:
+
+About mods using PCK files
+--------------------------
+
+While it's easy enough to create mods for a Godot game using PCK files, **we don't recommend doing it**.
+
+The reason is simple: :ref:`PCK files are very powerful <doc_exporting_pcks_overview_of_pck_files>`. They can override and replace content in a Godot game.
+While this seems ideal for modding purposes (and yes, it is), it comes with the downside of giving the keys to the castle to any bad actor.
+
+While replacing shaders and models is quite benign, replacing scripts is problematic, as the GDScript scripting engine doesn't have actually
+any concept of a `sandbox <https://en.wikipedia.org/wiki/Sandbox_(computer_security)>`__. Every mod containing GDScript files have access
+to the entire Godot Engine API, including making network calls and accessing the filesystem.
+
+We do currently encourage game developers to create their own modding system for the time being. To be really secure, any modding system
```

Comment:
```suggestion
We do currently encourage game developers to create their own modding system. To be really secure, any modding system
```
No need to specify both "currently" and "for the time being".

We *could* also mention the criteria for no longer recommending this (presumably some implementation of sandboxing?), but I think that doing so is optional.